### ISSUE TITLE
Menu: should Alt move focus into the top level menu items?

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#52936]
Refs: #52854

We might need an accessible way to put focus into the menu bar so that it can be explored by a screen reader. The native behaviour seems to be to move focus to the first top level menu entry when you press `Alt` and release it.

**Comment by OSadovy** [comment]
this does not work for me. Using NVDA 2018.2.1, vs code 1.26.1 - made sure use menu key mnemonics setting is set to on. pressing left alt key does not make NVDA to speak anything. using arrow keys to navigate moves me inside the editor.

**Comment by OSadovy** [comment]
well, seems it started to work after I changed ```"window.titleBarStyle": "custom",``` - there perhaps should be mention in the documentation somewhere about it.

**Comment by sbatten** [comment]
@OSadovy thanks for verifying with the new titlebar style. Currently, the new style is still in preview and disabled by default, but we hope to change that soon :)
